core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt

```diff
@@ -38,13 +38,12 @@ abstract class ToolkitCredentialsProvider : AwsCredentialsProvider {
      * Returns true or throws an Exception
      */
     @Throws(Exception::class)
-    open fun isValidOrThrow(stsClient: StsClient) {
+    open fun getAwsAccountOrThrow(stsClient: StsClient): String =
```

Comment:
I think we can drop the "OrThrow", its kinda an anti-pattern in naming

Reply:
Done.